spec/models/category_spec.rb

```diff
@@ -1,5 +1,32 @@
 require 'rails_helper'
 
 RSpec.describe Category, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'validations' do
+    it 'is valid with a name and image' do
+      category = Category.new(name: 'Fruits', image: Rack::Test::UploadedFile.new(Rails.root.join('spec/fixtures/images/apple.jpg'), 'image/jpg'))
```

Comment:
Well done writing unit tests for the models and routes 💯 

[OPTIONAL] I think it would be best if you added integration tests for all the most important components of the RoR application like the `login and registration` pages as well as your `views` as per project requirements using the `Capybara` gem.

Reply:
- [ ] [OPTIONAL] Just a reminder to ensure you add integration tests for all the components as mentioned initially by the previous reviewer.